### ISSUE TITLE
[BUG]: Compiler warnings not shown in compiler Output

### ISSUE THREAD

**Issue by carlosgalvezp** [compiler-explorer/compiler-explorer#6870]
### Describe the bug

Compiler warnings are not showing up in the compiler output. They do show up in the code, when one hovers the mouse over them.

### Steps to reproduce

1. Visit https://godbolt.org/z/GMsWbxcq5
2. Notice how the Output tab just says "Compiler returned: 0"
3. Hover over the ".a = 123" part
4. Read the warning

### Expected behavior

The Output tab shows:

```
foo.cpp: In function ‘int main()’:
foo.cpp:9:11: warning: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Wc++20-extensions]
    9 |     Foo f{.a = 123, .b = 321};
      |           ^
foo.cpp:9:21: warning: C++ designated initializers only available with ‘-std=c++20’ or ‘-std=gnu++20’ [-Wc++20-extensions]
    9 |     Foo f{.a = 123, .b = 321};
      |                     ^
```

### Reproduction link

https://godbolt.org/z/GMsWbxcq5

### Screenshots

![Screenshot from 2024-09-19 11-02-48](https://github.com/user-attachments/assets/b2767975-0187-4cf8-bd73-7888ecebb2f1)

**Comment by partouf** [comment]
they do show up, but you have the output open for gcc 11.2 here and not for 13.3 that gives the warnings

**Comment by carlosgalvezp** [comment]
Oh wow that was such a miss, thanks for pointing it out and sorry for the false alarm!
